README.md

```diff
@@ -40,7 +40,7 @@ If you want to compile yourself from the sources, you need a working Go
 setup. Then use the provided Makefile (type `make`).
 
 For the most basic setup, just start the binary. To change the address
-to listen on, use the `-web.listen-address` flag. By default, Pushgateway 
+to listen on, use the `-web.listen-address` flag (ie "0.0.0.0:9091" or ":9091"). By default, Pushgateway
```

Comment:
Nit: Please write "i.e." and re-wrap the lines to 80 chars for consistency with the rest of the file.

Reply:
Isn't this an e.g. rather than an i.e.? You can use values other than these.

Reply:
Oops, yeah, @brian-brazil is right, this should be an "e.g.". And sure, other examples than the default might make more sense, but I don't feel strongly…

Reply:
To clarify: @brian-brazil wasn't suggesting to actually use other values but only tried to explain why it is "e.g." rather than "i.e.".

So @maurorappa , my apologies for the 2nd-order nit, but if you could just change "i.e." to "e.g.", we are good to go.